packages/client/package.json

```diff
@@ -5,14 +5,15 @@
   "description": "Experimental email client with social features",
   "author": "Jesse Hallett <jesse@sitr.us>",
   "license": "Apache-2.0",
+  "homepage": "./",
```

Comment:
This `homepage` setting is how you configure create-react-app to serve assets from a non-root host location.